packages/design-system/src/styles/settings/variables/_build.scss

```diff
@@ -8,3 +8,5 @@ $ds-include-base-html-rulesets: true !default;
 
 // Set $asset-pipeline to true if you're using the Rails Asset Pipeline
 $asset-pipeline: false !default;
```

Comment:
Not related to focus styles, but we can probably remove this $asset-pipeline variable. I dont think we were really supporting it to begin with, it might have been leftover from uswds

Reply:
@line47 small clean up here